Dokument/Dateien/Kultur.tex

```diff
@@ -6,31 +6,51 @@ \subsection{Zwerge}
+Auch in der Familie zieht sich das ebenbürtige Bild von Frau und Mann durch. So werden die Aufgaben im Haushalt je nach den Stärken aufgeteilt (bspw. kann ein Mann vermutlich besser Wäsche waschen und eine Frau besser haushalten). 
+Die Aufteilung der Aufgaben ist dabei sogar notwendig, da beide gleichermaßen ins Arbeitsleben eingespannt sein können. 
+Es werden übrigens nur monogame Beziehungen geduldet.
 \\
 \\
-Ebenso verhält es sich mit der Kindererziehung. Je nach den täglichen Tätigkeiten der Eltern können diese mitgenommen und entsprechend erzogen und aufgezogen werden. So werden die beruflichen Fertigkeiten und allgemeinen Qualitäten direkt an die Kinder weitergegeben.
+Ebenso verhält es sich mit der Kindererziehung. 
+Je nach den täglichen Tätigkeiten der Eltern können diese mitgenommen und entsprechend erzogen und aufgezogen werden. 
+So werden die beruflichen Fertigkeiten und allgemeinen Qualitäten direkt an die Kinder weitergegeben.
 
-Als Säuglinge werden die Kinder im Idealfall von noch lebenden, aber nicht mehr arbeitstüchtigen Großeltern betreut. Jedoch nur solange, bis die Kinder in der Lage sind, ihre Eltern beim Tagesgeschäft zu begleiten.
+Als Säuglinge werden die Kinder im Idealfall von noch lebenden, aber nicht mehr arbeitstüchtigen Großeltern betreut. 
+Jedoch nur solange, bis die Kinder in der Lage sind, ihre Eltern beim Tagesgeschäft zu begleiten.
 \\
 \\
 \textbf{Exkurs: Homosexualität}\\
-Homosexualität ist kein Verbrechen und wird auch nicht als unnatürlich angesehen. Will ein homosexuelles Pärchen eine Lebensgemeinschaft eingehen, dann verpflichten sie sich damit automatisch Kinder aufzunehmen, die von Ihrer eigentlichen Familie nicht mehr versorgt werden können oder die keine Familie mehr haben.
+Homosexualität ist kein Verbrechen und wird auch nicht als unnatürlich angesehen. 
+Will ein homosexuelles Pärchen eine Lebensgemeinschaft eingehen, dann verpflichten sie sich damit automatisch Kinder aufzunehmen, die von Ihrer eigentlichen Familie nicht mehr versorgt werden können oder die keine Familie mehr haben.
 
 \subsection{Alter}
-Wie im vorhergehenden Punkt beschrieben, können ältere Menschen bei der Kinderbetreuung und -erziehung sehr hilfreich sein. Aus diesem Grund sind sie recht gut angesehen. Außerdem können sie wertvolle Erfahrungen mit Jüngeren teilen. Obwohl sie also keine Arbeit mehr leisten können, tragen sie positiv zur Gesellschaft bei.
+Wie im vorhergehenden Punkt beschrieben, können ältere Menschen bei der Kinderbetreuung und -erziehung sehr hilfreich sein. 
+Aus diesem Grund sind sie recht gut angesehen. Außerdem können sie wertvolle Erfahrungen mit Jüngeren teilen. 
+Obwohl sie also keine Arbeit mehr leisten können, tragen sie positiv zur Gesellschaft bei.
```

Comment:
Warum kann z.B. ein kampferprobter Heerführer in gehobenem Alter, der noch all seine Gliedmaßen besitzt, keine Arbeit mehr verrichten? Er ist vielleicht nicht mehr in seiner körperlichen Blüte, steht im Vergleich mit anderen doch aber immer noch gut da....